.circleci/install_deps.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 set -e
 apt-get update
-apt-get install -y libgtest-dev g++ make cmake wget build-essential clang clang-tidy-3.8
+apt-get install -y libgtest-dev g++ make cmake wget build-essential clang clang-tidy-3.8 libgtest-dev
```

Comment:
please remove this line, `libgtest-dev` is already there ;)

Reply:
Yes, removed it in an earlier PR#36